{
  "uid" : "cbd92e96b78564f8",
  "name" : "test_002_username_apply_permission",
  "fullName" : "testcases.高危命令.pg.test_004_table_group_pg#test_002_username_apply_permission",
  "historyId" : "2f5f96448105550cfb5750d8ba354e14",
  "time" : {
    "start" : 1700200892979,
    "stop" : 1700200893063,
    "duration" : 84
  },
  "description" : "\n    # 普通用户高危提权成功\n    此时此用户进行高危提权（insert,updata,delete,creat_table）成功，断言可以执行通过\n    :return:\n    ",
  "descriptionHtml" : "<pre><code># 普通用户高危提权成功\n此时此用户进行高危提权（insert,updata,delete,creat_table）成功，断言可以执行通过\n:return:\n</code></pre>\n",
  "status" : "broken",
  "statusMessage" : "TypeError: 'bool' object is not subscriptable",
  "statusTrace" : "def test_002_username_apply_permission():\n        \"\"\"\n        # 普通用户高危提权成功\n        此时此用户进行高危提权（insert,updata,delete,creat_table）成功，断言可以执行通过\n        :return:\n        \"\"\"\n        _sql_list = list(pg_database_sql.values())\n>       _info = apply_flow.high_risk_flow(_connection, username)\n\ntestcases\\高危命令\\pg\\test_004_table_group_pg.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntins\\api\\flow\\apply_flow.py:351: in high_risk_flow\n    \"connectionId\": public.get_connection_id(connection_name),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nconnection_name = 'xyz_test6666_PostgreSQL'\n\n    def get_connection_id(connection_name):\n        \"\"\"\n        通过连接名称获取connection_id\n        :param connection_name:\n        :return:\n        \"\"\"\n        _info = get_info()\n>       connection_id = jsonpath.jsonpath(_info, \"$.data.data[?(@.nodeName=='{}')].connectionId\".format(connection_name))[0]\nE       TypeError: 'bool' object is not subscriptable\n\ntins\\api\\public.py:317: TypeError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "description" : "\n    # 普通用户高危提权成功\n    此时此用户进行高危提权（insert,updata,delete,creat_table）成功，断言可以执行通过\n    :return:\n    ",
    "status" : "broken",
    "statusMessage" : "TypeError: 'bool' object is not subscriptable",
    "statusTrace" : "def test_002_username_apply_permission():\n        \"\"\"\n        # 普通用户高危提权成功\n        此时此用户进行高危提权（insert,updata,delete,creat_table）成功，断言可以执行通过\n        :return:\n        \"\"\"\n        _sql_list = list(pg_database_sql.values())\n>       _info = apply_flow.high_risk_flow(_connection, username)\n\ntestcases\\高危命令\\pg\\test_004_table_group_pg.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntins\\api\\flow\\apply_flow.py:351: in high_risk_flow\n    \"connectionId\": public.get_connection_id(connection_name),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nconnection_name = 'xyz_test6666_PostgreSQL'\n\n    def get_connection_id(connection_name):\n        \"\"\"\n        通过连接名称获取connection_id\n        :param connection_name:\n        :return:\n        \"\"\"\n        _info = get_info()\n>       connection_id = jsonpath.jsonpath(_info, \"$.data.data[?(@.nodeName=='{}')].connectionId\".format(connection_name))[0]\nE       TypeError: 'bool' object is not subscriptable\n\ntins\\api\\public.py:317: TypeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "fa873dab46cc4fc0",
      "name" : "log",
      "source" : "fa873dab46cc4fc0.txt",
      "type" : "text/plain",
      "size" : 907
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 0
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "testcases.高危命令.pg"
  }, {
    "name" : "suite",
    "value" : "test_004_table_group_pg"
  }, {
    "name" : "host",
    "value" : "LAPTOP-BQ91TRPV"
  }, {
    "name" : "thread",
    "value" : "27328-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcases.高危命令.pg.test_004_table_group_pg"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "cbd92e96b78564f8.json",
  "parameterValues" : [ ]
}